extension/config/jest/setupTests.ts

```diff
@@ -0,0 +1,52 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import { JSDOM } from "jsdom";
+import fetch from "isomorphic-unfetch";
+
+// make a JSDOM thing so we can fuck with mount
+const jsdom = new JSDOM("<!doctype html><html><body></body></html>");
+const { window } = jsdom;
```

Comment:
This file is mostly copied from an old project (converted to TS). Nice to have on hand so we don't need to re-solve these environment problems, stuff like global mocks and overrides

Reply:
it'd be nice to either make this a separate npm package that we maintain or use an existing package that does this (esp the storage stuff https://www.npmjs.com/package/jest-localstorage-mock). Would rather not have to maintain all this setup stuff

Reply:
Ah fair point. I'll see if there are good package for these mocks, that one looks solid. I'm wary of stuff like that being too heavy or complex

Reply:
Updated @piyalbasu!